### PR TITLE
Run CI tests using Gihub actions (on 9.5)

### PR DESCRIPTION
**Pull request by cedric-anne**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

Port of #6366 for 9.5 branch.

### CODE REVIEW

**Comment by cedric-anne** [comment]
Have to find a way to make imap tests use the good host.

File: bin/console [hunk 1]
```diff
@@ -48,7 +48,7 @@ if (isset($_SERVER['argv']) && ['dependencies', 'install'] === array_slice($_SER
 
    if (array_key_exists('ci', $options) && true === $options['ci']) {
       $composer_command .= ' --prefer-dist --no-progress';
-      $npm_command = 'npm ci';
+      //$npm_command = 'npm ci';
```
Comment (trasher) [inline bin/console]: You should drop or uncomment :) (dunno why you commented, nor what is the original purpose).

**Comment by trasher** [comment]
Rely on https://github.com/glpi-project/glpi/pull/6366

File: bin/console (hunk 1 above)
Comment (cedric-anne) [reply bin/console]: Purpose was to not alter package-lock.json on install, but it is not important anymore as we changed way to check if dependencies are up to date.

**Comment by cedric-anne** [comment]
To pass PHP 7.4-rc test suite, I had to remove some type hinting (see https://github.com/glpi-project/glpi/pull/6367/commits/9cc4e5b3137ed0f0618824b053d024ecafb509dc ).

**Comment by cedric-anne** [comment]
> To pass PHP 7.4-rc test suite, I had to remove some type hinting (see [9cc4e5b](https://github.com/glpi-project/glpi/commit/9cc4e5b3137ed0f0618824b053d024ecafb509dc) ).

I added a patch for atoum (applied on composer install/update) to be able to revert this point.

**Comment by trasher** [comment]
Is there a PR for atoum fix?

**Comment by cedric-anne** [comment]
https://github.com/atoum/atoum/pull/827

**Pull request merged**